lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -1681,6 +1692,147 @@ def test_is_content_authored_by(self):
         self.assertFalse(utils.is_content_authored_by(content, user))
 
 
+class GroupModeratorPermissionsTestCase(ModuleStoreTestCase):
+    """Test utils functionality related to forums "abilities" (permissions) for group moderators"""
+
+    def _check_condition(user, condition, content):
```

Comment:
When writing the test I found that is_open and is_team_member_if_applicable were always false. I couldn't find any usages of setting a discussion to open that weren't mocking out API calls (which this test doesn't do, so it felt out of place to add it just to have the right start state for the actual test). Likewise, the other test (django_comment_client/base/tests.py) added users to a team and checked for the functionality there, so it seemed that it would be duplicating those tests a little to also go through adding all the users to a team, since we're not actually looking for anything related to editing privileges outside of a team in this test at all.